README.md

```diff
@@ -116,11 +116,15 @@ To run the project with Visual Studio you can open the Solution (`.sln`) file an
 
 #### Running With Visual Studio Code
 
-To run the project with Visual Studio Code you will first need to open the repository directory with Visual Studio Code using the `Open Folder...` option.
+In Visual Studio Code, open the workspace by choosing `File` > `Open Folder...` and selecting the repository directory.
 
-Second, you need to [install the recommended extensions for the workspace](https://code.visualstudio.com/docs/editor/extension-gallery#_recommended-extensions). Note that extension recommendations are classified as either "Workspace Recommendations" or "Other Recommendations", but only the "Workspace Recommendations" are required.
+Next, [install the recommended extensions for the workspace](https://code.visualstudio.com/docs/editor/extension-gallery#_recommended-extensions). When you first open the workspace, Visual Studio Code may prompt you to install recommended extensions. Another way to find these is to navigate to `View` > `Extensions`, and then type `@recommended` in the filter. Note that extension recommendations are classified as either "Workspace Recommendations" or "Other Recommendations", but only the "Workspace Recommendations" are required. If you don't already have the [.NET SDK](https://dotnet.microsoft.com/en-us/download) installed, the extension setup will guide you through installing it.
 
-After the required extensions are installed, you can run the server by pressing `F5`.
+Build the project by choosing `Terminal` > `Run Build Task...`. The integrated terminal will appear and show the progress of the build. After it completes, press any key to close the terminal.
+
+Place the [web client static files](#installing-the-web-client) into the build output directory. The initial build will have created a directory in the repository at path `Jellyfin.Server/bin/Debug/net8.0` (or similar, based on your SDK version). Place the web client static files in a subdirectory called `jellyfin-web`.
```

Comment:
While there is a launch config to run without the web client, it's not currently the default launch config. What ends up happening for a first time user is they press F5 to launch, the server builds, and then terminates with an error on the console:

```
[15:28:37] [ERR] [1] Main: The server is expected to host the web client, but the provided content directory is either invalid or empty: .../Jellyfin.Server/bin/Debug/net8.0/jellyfin-web. If you do not want to host the web client with the server, you may set the '--nowebclient' command line flag, or set'hostwebclient=false' in your config settings
The program '[60385] jellyfin.dll' has exited with code 0 (0x0).
```

The error message might not even be apparent in the terminal, since its amongst dozens of other messages, and focus can move from the Terminal and Debug Console panels.

In my opinion, the current choice of the default launch config is correct. I think most developers will want to start with a web client to actually interact with the server. If that's not what we intend, then I think the section above about installing the web client should also be moved a bit lower to get out of the way.

Also, I agree that we should let people know the launch config without the web client is available. However, I don't know if this README is the best place for describing anything besides the default. I think this space is best used as a quick start, and there's an opportunity for deeper developer documentation to describe other options. Besides, once you're up and running, its much easier to discover the other launch configuration.

Reply:
Well i see the point however this is still the server, not the combined "install" of it. I use another hosted instance to connect to the server to not overcomplicate things for example. 

That its not the default is correct and i think its still a mistake but nobody bothered because its dev stuff. I think we can leave the "whats normal users expect" aspect out of focus here, devs need to adjust to a lot of stuff in the JF project, the whole distinction to start with or without is always the first start.
I would just put it as the default option as you already touched the topic of configuration and build scripting and would go in this route:

> Next is to evaluate if you want to start just the server or do you want to start the server and the web UI. If you only intend to run the server, change the Lunch config in the VScode Debug tab
![image](https://github.com/jellyfin/jellyfin/assets/6794763/97ae5b91-1fb1-46c6-a547-907781392514)
> To `.NET Launch (nowebclient)` if you only want to lunch the server.
> If you want to lunch with the webclient ... [INSERT YOUR SECTION]

Reply:
Also i expect any reputable dev to see that log line, knowing what the console window is and being capable of making a full text search or just reading the docs on that matter.

Reply:
Perhaps we aren't currently aligned on the audience for the README. In my opinion, the README is for new developers to the project, new developers to C# and the specific technologies used in jellyfin, and even new developers in general. This is the "welcome mat" inviting developers to step inside. Earlier I used the word user, which was a mistake - I don't think the README is for end users of jellyfin.

With this in mind, I think the fewer decisions they need to make up front, the quicker they can get going. The less we assume about their expertise, the better. How would a new developer know what the limits of nowebclient versus console even are? This is why I think its better to start with one way of launching, and then use other developer docs (which currently don't seem to exist, but we can work on that later) to further elaborate on the options. If we agree that less decisions is better, then which should be the default?

I believe most new developers will benefit from starting the server with a web client on their first run. Without the web client, there's actually very little you can do. I imagine the most common first steps would be to walk through the server setup in the browser in the least. Otherwise if you started without the web client on your first run as someone new to the project, there's very little you could actually do - I'd love to hear any examples you might have.

Reply:
I use the `--webdir` argument in `launch.json` to specify the webui path. It might be a good idea to mention this for newcomers, especially if they are not going to build webui.

Reply:
Yup, I'm agreeing with you. I just don't think that mention should be in the README. I hope we can land this change and then add details like the `--webdir` argument in a subsequent PR for more detailed developer docs. Can we agree that this is a step in the right direction?

Reply:
Resolved @dmitrylyzo's concern.